{"name": "测试用例005：用户名存在特殊字符", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 5", "trace": "@allure.feature('测试需求001，用户注册')\n    @allure.story('测试场景002，微信用户注册校验')\n    @allure.title('测试用例005：用户名存在特殊字符')\n    def test_case05():\n>       assert 1==5\nE       assert 1 == 5\n\ntest_allure.py:38: AssertionError"}, "start": 1619764126705, "stop": 1619764126705, "uuid": "66576c35-beab-4f5d-9320-9189153f2f55", "historyId": "7f3ec14765b36cbd09bed0216d93051f", "testCaseId": "3962cfd485cf703c70d8b8f2c2f6d2bf", "fullName": "test_allure#test_case05", "labels": [{"name": "story", "value": "测试场景002，微信用户注册校验"}, {"name": "feature", "value": "测试需求001，用户注册"}, {"name": "suite", "value": "test_allure"}, {"name": "host", "value": "DESKTOP-ATELKEB"}, {"name": "thread", "value": "20680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}